api/src/main/java/openconsensus/trace/export/ExportComponent.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2017, OpenConsensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package openconsensus.trace.export;
+
+import openconsensus.trace.data.TraceOptions;
+
+/**
+ * Class that holds the implementation instances for {@link SpanExporter}.
+ *
+ * <p>Unless otherwise noted all methods (on component) results are cacheable.
+ *
+ * @since 0.5
+ */
+public abstract class ExportComponent {
```

Comment:
This is not needed, probably we should have a "recordSpanData" API at the Tracer level.

Reply:
You need something to record `SpanData` to. If not this - we need `Tracer` to accept `SpanData`

Reply:
yes, I was thinking on having `Tracer.recordSpanData(mySpanData)` (probably a more suggestive name is better).

Reply:
This would be fine with me. In fact, I think it is better for consitency.